albumentations/augmentations/geometric/transforms.py

```diff
@@ -150,6 +150,7 @@ def __init__(
         mask_value=None,
         always_apply=False,
         approximate=False,
+        same_dxdy=False,
```

Comment:
Please, add this variable into the docstring with a description. And add it into `get_transform_init_args_names`

Reply:
Done!